msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/PermissionLevelManagerImpl.java

```diff
@@ -913,6 +921,7 @@ private PermissionsMask getDefaultReviewerPermissionsMask() {
 		  mask.put(PermissionLevel.READ, Boolean.valueOf(true));
 		  mask.put(PermissionLevel.MARK_AS_READ,Boolean.valueOf(true));
 		  mask.put(PermissionLevel.MODERATE_POSTINGS, Boolean.valueOf(false));
+		  mask.put(PermissionLevel.IDENTIFY_ANON_AUTHORS, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.DELETE_OWN, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.DELETE_ANY, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.REVISE_OWN, Boolean.valueOf(false));
```

Comment:
Like my earlier comment, the above code seems to be duplicated a lot. Opportunity for a refactor?